src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/ObjectWriter.cs

```diff
@@ -767,9 +784,10 @@ public static void EmitObject(string objectFilePath, IEnumerable<DependencyNode>
                 }
                 else
                 {
-                    objectWriter.SetSection(MethodCodeNode.UnixContentSection);
+                    managedCodeSection = MethodCodeNode.UnixContentSection;
                     objectWriter.SetSection(LsdaSection);
                 }
+                objectWriter.SetCodeSectionAttribute(managedCodeSection);
```

Comment:
Should Windows sentinels  have executable attribute? Do they contain instructions?

Should .text section be IsStandardSection, if it has executable  attribute set on and we don't pass it right now.